security/pkg/stsservice/tokenmanager/google/tokenexchangeplugin.go

```diff
@@ -0,0 +1,321 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package google
+
+import (
+	"bytes"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"sync"
+	"time"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/golang/protobuf/ptypes/duration"
+
+	"istio.io/istio/security/pkg/stsservice"
+	"istio.io/pkg/log"
+)
+
+const (
+	httpTimeOutInSec = 5
+	maxRequestRetry  = 5
+	contentType      = "application/json"
+	scope            = "https://www.googleapis.com/auth/cloud-platform"
+	tokenType        = "urn:ietf:params:oauth:token-type:access_token"
+	federatedToken   = "federated token"
+	accessToken      = "access token"
+)
+
+var (
+	pluginLog              = log.RegisterScope("tokenManagerGooglePlugin", "token manager plugin debugging", 0)
```

Comment:
```suggestion
	pluginLog              = log.RegisterScope("token", "token manager plugin debugging", 0)
```

long scope has a major impact on log readability

Reply:
Thanks for pointing this out.